### PR TITLE
bots, test: Introduce rhel-8-1-distropkg mode

### PR DESCRIPTION
**Pull request by martinpitt**
We will use this to test the rhel-8-appstream branch as well as z-stream
updates. This will also ensure that we don't break the wire protocol.

 - [x] Add rhel-8-1-distropkg test skips

### CODE REVIEW

**Comment by martinpitt** [comment]
@Gundersanne: Does [this failure](https://209.132.184.41:8493/logs/pull-12735-20190909-103333-5d9110e3-cockpit-project-cockpit--rhel-8-1-distropkg/backup.Ot8O5V/log.html#196-2) ring a bell? When I replace `.listing-ct-actions` with `.listing-ct`, it works. But I didn't see an obvious change of that between 196 and master?

After changing that locally, it still fails on
```
  File "test/verify/check-networking-firewall", line 406, in removeZone
    b.wait_not_present("#zones-listing tr[data-row-id='{}']".format(zone))
```

Do you have some time to take a look here?

**Comment by Gundersanne** [comment]
@martinpitt I think this should fix it: https://github.com/Gundersanne/cockpit/commit/9d0ca510e5683cb2ad8d8d0a58d1c986573062d1

File: bots/images/scripts/rhel-8-1-distropkg.install [hunk 1]
```diff
@@ -0,0 +1,5 @@
+#! /bin/bash
```
Comment (marusak) [inline bots/images/scripts/rhel-8-1-distropkg.install]: We should delete then `bots/images/scripts/rhel-8-0-distropkg.install`

File: test/verify/check-active-pages [hunk 2]
```diff
@@ -30,7 +30,7 @@ class TestActivePages(MachineCase):
         self.login_and_go("/system")
 
         # Preloading of playground/preloaded only happens with #12534
-        if m.image in [ "rhel-8-0-distropkg" ]:
```
Comment (marusak) [inline test/verify/check-active-pages]: Is there any need to keep `rhel-8-0-distropkg` here?
I am not sure what exactly we want - whether to keep `rhel-8-0-distropkg` but just not to run it automatically or to drop it completely. On one hand you removed it from testmap, so we cannot trigger it and you also removed it from doc, on the other hand you kept the install script and keep it also in tests.

Comment (martinpitt) [reply test/verify/check-active-pages]: Right, I wanted to get this to green, land it, disable 8-0-distropkg, and as a follow-up remove all the references to it an the rhel-8-0 image. I just didn't want to conflate the two.

**Comment by martinpitt** [comment]
@marusak : I now added cleanup of 8.0 and rearranged the commits a bit. Can you please have another look?

File: bots/image-refresh [hunk 3]
```diff
@@ -100,11 +100,11 @@ TRIGGERS = {
         "rhel-7-8@cockpit-project/cockpit/rhel-7.8",
     ],
     "rhel-8-0": [
-        "rhel-8-0",
-        "rhel-8-0-distropkg",
+        "rhel-8-0@cockpit-project/cockpit-/rhel-8.0",
```
Comment (marusak) [inline bots/image-refresh]: `rhel-8-0@cockpit-project/cockpit/rhel-8.0` :)

Comment (martinpitt) [reply bots/image-refresh]: Bah, silly me -- fixed!

File: test/verify/check-realms [hunk 4]
```diff
@@ -257,12 +243,7 @@ class TestRealms(MachineCase):
         b.click("#system-info-domain a")
         b.wait_popup("realms-op")
         b.set_val(self.op_address, "NOPE")
-        if m.image not in ["rhel-8-0-distropkg"]:
-            # changed in PR #11022
-            b.wait_text(".realms-op-address-error", "Domain NOPE could not be contacted")
-            b.wait_not_visible(".realms-op-address-spinner")
-        else:
-            b.wait_attr(".realms-op-address-error", "title", "Domain NOPE could not be contacted")
+        b.wait_attr(".realms-op-address-error", "title", "Domain NOPE could not be contacted")
```
Comment (marusak) [inline test/verify/check-realms]: You kept the wrong branch here :) (https://209.132.184.41:8493/logs/pull-12735-20190910-070707-23002eae-cockpit-project-cockpit--rhel-8-1-distropkg/log.html#92-2)

Comment (martinpitt) [reply test/verify/check-realms]: Argh, lots of `in` vs `not in`.. fixed.

**Comment by martinpitt** [comment]
rhel-8-0-distropkg failure is expected, as this PR just takes this away. debian-testing failure is unrelated.

**Pull request merged**